src/utils/Util.ts

```diff
@@ -277,6 +277,16 @@ export class Util {
     }
   }
 
+  async createProgressBar(
+    formattedCurrentTime: string,
+    formattedDuration: string | undefined,
+    options = { length: 15, indicator: "🔘", line: "▬" },
+  ) {
+    return `${formattedCurrentTime} | ${options.indicator}${options.line.repeat(
```

Comment:
this would never actually work, correct? The indicator will always show at the start of the line

Reply:
Yes, It's a preview and not something that actually will function as a progressBar. Though some small changes would make that possible.

Reply:
I think it would look better if it’s functional.

Reply:
alright, will merge this this week and try implement this myself.